Makefile

```
@@ -68,7 +68,9 @@ NO_VALUE ?= no_value
 #		VARIABLES
 ###############################
 PWD ?=  $(shell pwd)
-VERSION ?= $(shell git describe --tags --abbrev=0)
+ifndef VERSION
+VERSION := $(shell git fetch -f --tags &> /dev/null ; git describe --tags --abbrev=0)
```

just a small one: If we use verbose flag variants (`--tags` instead of `-t`), it makes sense to make the `force` option verbose as well (`--force` instead of `-f`)
```suggestion
VERSION := $(shell git fetch --force --tags &> /dev/null ; git describe --tags --abbrev=0)
```

---

ammended & 'push-f'ed